package.json

```diff
@@ -1,22 +1,24 @@
 {
   "name": "jsdoc-template",
```

Comment:
Do we consider this template open source? Should we give it a name and publish it to npm? 

We could also publish it under the `@braintree` namespace.